catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/InputTransformerIds.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.content.monitor;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableSet;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class InputTransformerIds {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(InputTransformerIds.class);
+
+  private static Path transformerFolder = Paths.get("etc", "transformers");
```

Comment:
❗️ Why are we adding json filesystem parsing instead of just using a metatype config?

Reply:
I did it this way so downstream projects could drop json files with only their input transformer IDs.

Reply:
See https://github.com/codice/alliance/pull/672

Reply:
Still seems like metatypes would work. Each distro should be responsible for what transformers it cares about.

Reply:
They would, but I took the approach that each distribution would define its set of InputTransformers and build on each other. I also wanted to avoid exposing this list to administrators until we have a need for it.